### ISSUE TITLE
FEV1 Internal Validation: FEV1 decline not zero even after setting Zafar's coefficient's as zero

### ISSUE THREAD

**Issue by aminadibi** [aminadibi/epicR#57]
problem seems to come from the second line in:
```
       (*ag).fev1 = (*ag).fev1_baseline
                + 0*(*ag).fev1_slope*(*ag).followup_time
                + input.lung_function.fev1_betas_by_sex[7][(*ag).sex]*(*ag).followup_time*(*ag).followup_time;  

```

**Comment by aminadibi** [comment]
problem seems to lie in the first line of:
```
      (*ag).fev1_slope=  0*(*ag).fev1_decline_intercept
      +input.lung_function.fev1_betas_by_sex[1][(*ag).sex]*(*ag).age_baseline
      +input.lung_function.fev1_betas_by_sex[2][(*ag).sex]*(*ag).weight_baseline
      +input.lung_function.fev1_betas_by_sex[3][(*ag).sex]*(*ag).height
      +input.lung_function.fev1_betas_by_sex[4][(*ag).sex]*(*ag).height*(*ag).height
      +input.lung_function.fev1_betas_by_sex[5][(*ag).sex]*(*ag).smoking_status
      +input.lung_function.fev1_betas_by_sex[6][(*ag).sex]*(*ag).age_baseline*(*ag).height*(*ag).height;
```

**Comment by aminadibi** [comment]
things that had to change for us to get zero FEV1 decline:
* disabling `FEV1_tail` traps (2 instances)
* smoking change only allowed when COPD is not present (2 instances)
* zeroing `(*ag).fev1_decline_intercept` in event_process (not in create_agent)
* zeroing fev1 decline predictors in `input$lung_function$fev1_betas_by_sex`

**Comment by aminadibi** [comment]
Fixed in [v0.17.0](https://github.com/aminadibi/epicR/releases/tag/v0.17.0)
